multicluster/service-mirror/cluster_watcher.go

```diff
@@ -682,6 +698,16 @@ func (rcsw *RemoteClusterServiceWatcher) getMirrorServices() ([]*corev1.Service,
 	if err != nil {
 		return nil, err
 	}
+	if len(services) == 0 {
```

Comment:
Is a check `kerrors.IsNotFound(err)` also required here, or does `List()` just ensure the `services` will be an empty array in this case?

Reply:
If there are no matching services, `services` will be an empty array and `err == nil`.